libsplinter/src/auth/oauth/rest_api/mod.rs

```diff
@@ -23,13 +23,13 @@ use crate::rest_api::{Resource, RestResourceProvider};
 
 use super::{OAuthClient, UserInfo};
 
-/// Saves a set of user info.
-pub trait SaveUserInfoOperation: Sync + Send {
+/// Operations that handle an auth user.
+pub trait AuthUserOperation: Sync + Send {
```

Comment:
Don't think I like this name; this trait isn't an operation, it's a storage API. It's also OAuth-specific. Maybe `OAuthUserStore`?

Reply:
`OAuthUserInfoStore`, as `OauthUserStore` is already a biome trait.  This store also acts on UserInfo structs, so it is probably better to be more specific.